src/features/show/selectors.js

```diff
@@ -553,7 +553,9 @@ export const getShowDurationAsString = createSelector(
 export const getShowDescription = createSelector(
   getNumberOfDronesInShow,
   getShowDurationAsString,
-  (numberDrones, duration) => `${numberDrones} drones, ${duration}`
+  getMaximumHorizontalDistanceFromTakeoffPositionInTrajectories,
```

Comment:
This is the _horizontal_ distance; we need a similar selector for vertical distances and then we need to use that here

Reply:
Actually, if the takeoff positions are at different altitudes, a similar selector would return an incorrect value; I guess we need a selector that returns the maximum Z coordinate across all trajectories and drones instead.